events.go

```diff
@@ -37,6 +37,14 @@ information is displayed once every 5 seconds.`,
 		if err != nil {
 			fatal(err)
 		}
+		id := context.Args().First()
```

Comment:
You can just use `container.ID()` here since it's already been loaded by factory.